pkg/scheduler/cache/cache.go

```diff
@@ -640,6 +640,13 @@ func newSchedulerCache(config *rest.Config, schedulerNames []string, defaultQueu
 		UpdateFunc: sc.UpdateNumaInfoV1alpha1,
 		DeleteFunc: sc.DeleteNumaInfoV1alpha1,
 	})
+	// explictly register informers to the factory, otherwise resources listers cannot get anything
```

Comment:
suggest move these register to line 489 and add comments to declare that it is for selectorSpread plugin.

Reply:
done